frontend/elements/example.css

```diff
@@ -249,7 +249,7 @@
 
 .hanko_exclamationMark {
     fill: var(--error-color, #e82020);
-    padding-right: 5px
+    padding-right: 1px 6px 0 4px
```

Comment:
Is this line correct? Shouldn't it be just "padding"?

Reply:
You are right, I forgot to update to just "padding" in the example.css